pkg/registry/core/event/strategy.go

```diff
@@ -71,7 +71,8 @@ func (eventStrategy) AllowCreateOnUpdate() bool {
 
 func (eventStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
 	event := obj.(*api.Event)
-	return validation.ValidateEvent(event)
+	oldEvent := obj.(*api.Event)
+	return validation.ValidateEventUpdate(event, oldEvent, api.SchemeGroupVersion)
```

Comment:
shouldn't this be extracting the request groupVersion from the context?

Reply:
Done.